lib/features/history/widgets/history_appbar.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app_kids/core/app/route_utils.dart';
+import 'package:givt_app_kids/shared/widgets/heading_2.dart';
+import 'package:go_router/go_router.dart';
+
+class HistoryAppBar extends StatelessWidget implements PreferredSizeWidget {
+  const HistoryAppBar({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return AppBar(
+      backgroundColor: const Color(0xFFEEEDE4),
+      elevation: 0,
+      title: const Row(
+        children: [
+          Heading2(text: 'My Givts'),
+          Spacer(),
+        ],
+      ),
+      leading: IconButton(
```

Comment:
we already have BackButton widget, that also tracks amplitude btw. Can we reuse it here?